nautobot/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nautobot-ui",
-  "version": "0.1.0",
+  "version": "2.0.0a1",
```

Comment:
We might want to add a `tasks.py` wrapper for `poetry version ...` that makes sure this gets updated in parallel when updating the version in `pyproject.toml`.